src/main/java/com/ss/training/utopia/agent/controller/AgentController.java

```diff
@@ -36,4 +38,67 @@
 		return new ResponseEntity<Booking[]>(bookingArray, status);
 	}
 
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+	@PutMapping(path="/booking")
+	public ResponseEntity<Booking> cancelBooking(@RequestBody Booking booking) {
+		HttpStatus status = HttpStatus.BAD_REQUEST;
+		if(booking == null || booking.getTravelerId() == null || booking.getBookerId() == null || booking.getStripeId() == null)
```

Comment:
It may not be necessary to check these things here (presumably we can make it so the frontend only passes valid bookings), but if we do then we should probably also check the flight ID and the active status.

Reply:
I am going to remove them.